src/renderer/components/watch-video-comments/watch-video-comments.vue

```diff
@@ -48,6 +48,7 @@
         <img
           :src="comment.authorThumb"
           class="commentThumbnail"
+          @click="goToChannel(comment.authorLink)"
         >
         <p class="commentAuthor">
```

Comment:
Please also make this clickable. Apply the same pointer CSS from the author thumbnail as well.